index.html

```diff
@@ -4409,6 +4409,33 @@ <h3>Verification</h3>
             </ol>
           </li>
           <li>
+Ensure that <var>result</var>.<var>document</var> meets all status
+expectations by performing the following steps for each
+<var>statusMechanism</var> associated with the
```

Comment:
If the `statusMechanism` is given an `@id`, a JSON-LD processor might discover more properties on it than a JSON processor would. Which set of properties gets used here?

Looking at https://www.w3.org/TR/vc-bitstring-status-list/#bitstringstatuslistentry, none of the properties are optional, so I think mischief is limited to providing extra copies of `statusListIndex` or `statusListCredential`. https://www.w3.org/TR/vc-bitstring-status-list/#validate-algorithm doesn't consider that there might be extra copies with different values ... or even that the required fields might be missing. I assume you'll want status checking to declare that the VC as a whole is invalid when any of these requirements are violated?